src/crypto/qat/qcccrypto.cc

```diff
@@ -39,7 +39,6 @@ static void symDpCallback(CpaCySymDpOpData *pOpData,
   }
 }
 
-static std::mutex qcc_alloc_mutex;
```

Comment:
I'd be interested in knowing why the author(s) thought we might need qcc_alloc_mutex

Reply:
use of that mutex was removed in https://github.com/ceph/ceph/pull/47040 because it could guarantee that all access happened on the same background thread